### ISSUE TITLE
Add Discord as a connector

### ISSUE THREAD

**Issue by rachfop** [onyx-dot-app/onyx#1179]
I saw that there was interest in making a Discord connector - but there wasn't an issue to track interest.
I'm adding this, so I can keep up to date on any progress made on this connector.

https://github.com/danswer-ai/danswer/pull/975

**Comment by NickSeagull** [comment]
Definitely, would love to have both @boostercloud and @neohaskell Discord servers connected to Danswer!

**Comment by pairofcrocs** [comment]
Can't wait for this :)

**Comment by IntrovertedFL** [comment]
Has there been any progress on this? Can't wait to use it :)

**Comment by voarsh2** [comment]
I would be interested in this for public facing Q/A for a product.... directing common product questions to DanswerAI might be useful, however, it would be great to only show relevant data via tags/defined categories for the Discord bot integration, for example.

**Comment by IntrovertedFL** [comment]
What's causing this to not move forward? I would be willing to "donate" a reasonable amount if that would push this along faster.

**Comment by ptiggens** [comment]
I would also be open to donating for this.

**Comment by IntrovertedFL** [comment]
There is a PR that was submitted last week, hopefully this will be approved soon https://github.com/danswer-ai/danswer/pull/3023

**Comment by cloudia-company** [comment]
it would be great

**Comment by paranoidcircut** [comment]
How does this open source thingy work? 
Does someone just go out and implement this?
